packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -97,6 +99,10 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
     }
   }
 
+  public componentWillUnmount(): void {
+    this._isUnmounted = true;
```

Comment:
I suppose you need `super.componentWillUnmount()` here.

Reply:
Good catch! I will try avoid using _isUnmounted first.